src/Transition/CreateNewIDFUsingRulesV8_5_0.f90

```diff
@@ -363,6 +365,49 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                 OutArgs(1) = sVersionNum
                 nodiff=.false.
 
+    !!!    Changes for this version
+              CASE('MATERIALPROPERTY:MOISTUREPENETRATIONDEPTH:SETTINGS')
```

Comment:
It looks like this merged okay, but it should probably be in a new `CreateNewIDFUsingRulesV8_6_0.f90` source file instead of here. In an effort not to screw things up, I'm going to leave that you @Myoldmopar.